skipchain/struct_test.go

```diff
@@ -183,6 +184,36 @@ func sign(msg SkipBlockID, servers []*onet.Server, l *onet.LocalTest) (*bftcosi.
 	return &bftcosi.BFTSignature{Sig: sig, Msg: msg, Exceptions: nil}, nil
 }
 
+func TestSkipBlock_GetFuzzy(t *testing.T) {
+	db, fname := setupSkipBlockDB(t)
```

Comment:
I think some more elaborate tests should be done:

```
list = {{1,2,3,6,4}, {2,3,4,7,5}}
search({1}) == 0
search({2}) == 1
search({3}) == 0
search({4}) == 0
search({5}) == 1
search({6}) == 0
search({7}) == 1
```
I think that should cover most of the test-cases.